crates/rover-client/src/blocking/client.rs

```diff
@@ -46,9 +47,12 @@ impl Client {
     ///
     /// If successful, it will return body.data, unwrapped
     pub fn handle_response<Q: graphql_client::GraphQLQuery>(
-        response: reqwest::blocking::Response,
+        response: Response,
     ) -> Result<Q::ResponseData, RoverClientError> {
-        let response_body: graphql_client::Response<Q::ResponseData> = response.json()?;
+        let response_text = response.text()?;
+        tracing::debug!("{}", &response_text);
```

Comment:
Can we move the rest of the tracing work on line 36 into here? So all the response tracing happens in one place